flexget/plugins/clients/flood.py

```diff
@@ -0,0 +1,442 @@
+                logger.debug(f"Successfully started {len(hashes)} torrents in Flood.")
+                return True
+
+            logger.debug(
+                f"Failed to start {len(hashes)} torrents in Flood. (status={response.status_code})"
+            )
+            return False
+        except RequestException as e:
+            raise PluginError(
+                f"An error occurred while attempting to start torrents in Flood: {e}"
+            )
+
+    def stop_torrents(self, config: dict, hashes: list) -> bool:
+        """
+        Stop a list of torrents in Flood.
+        Will only attempt to stop torrents if the JWT token is not expired or unset.
+        """
+
+        if not hashes:
+            raise PluginError("Parameter 'hashes' cannot be empty.")
+        if self.is_jwt_expired():
+            raise PluginError("Not authenticated with Flood.")
+
+        try:
+            response = post(
+                f"{config['url']}/api/torrents/stop",
+                json={"hashes": hashes},
+                cookies={"jwt": self.jwt},
+            )
+
+            if response.status_code == 200:
+                logger.debug(f"Successfully stopped {len(hashes)} torrents in Flood.")
+                return True
+
+            logger.debug(
+                f"Failed to stop {len(hashes)} torrents in Flood. (status={response.status_code})"
+            )
+            return False
+        except RequestException as e:
+            raise PluginError(f"An error occurred while attempting to stop torrents in Flood: {e}")
+
+    def delete_torrents(self, config: dict, hashes: list, deleteData: bool = False) -> bool:
+        """
+        Delete a list of torrents in Flood.
+        Will only attempt to delete torrents if the JWT token is not expired or unset.
+        """
+
+        if not hashes:
+            raise PluginError("Parameter 'hashes' cannot be empty.")
+        if self.is_jwt_expired():
+            raise PluginError("Not authenticated with Flood.")
+
+        try:
+            response = post(
+                f"{config['url']}/api/torrents/delete",
+                json={"hashes": hashes, "deleteData": deleteData},
+                cookies={"jwt": self.jwt},
+            )
+
+            if response.status_code == 200:
+                logger.debug(f"Successfully deleted {len(hashes)} torrents from Flood.")
+                return True
+
+            logger.debug(
+                f"Failed to delete {len(hashes)} torrents from Flood. (status={response.status_code})"
+            )
+            return False
+        except RequestException as e:
+            raise PluginError(
+                f"An error occurred while attempting to delete torrents from Flood: {e}"
+            )
+
+
+class InputFlood:
+    """Creates entries from torrents in Flood based
+
+    Example:
+        from_flood:
+            url: http://localhost:3000
+            username: username
+            password: password
+    """
```

Comment:
Maybe add descriptions for the config values? I know they're pretty self-explanatory but it lets people know what the options are.

Reply:
Added descriptions to module docstring examples in Winter/Flexget@c067577